deploy/helm/values.yaml

```diff
@@ -17,7 +17,7 @@ image:
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
   tag: ""
-  imagePullSecrets: []
+  imagePullSecrets: [ ]
```

Comment:
It's better not to automatically format this file, as different IDEs or developers may have different formats. It's better to adjust your IDE to adapt to the current format rather than vice versa.